### PR TITLE
Feature/create comments handlers

### PR DESCRIPTION
**Pull request by GMolicki**

### CODE REVIEW

File: comment.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	_, responseBody, _ := gorequest.New().Post(urlAddress).
+									Set(contentType, mediaTypeFormType).
+									Set(apiSignHeader, wh.hashRequest(urlAddress)).
+									End()
+
+	return wh.getResponseBodyAsCommentResponse(responseBody)
+}
+
+func getAddCommentUrl(linkId int, parentCommentId int, wh *WykopHandler) string {
+	methodWithParams := "add/" + strconv.Itoa(linkId)
```
Comment (agilob) [inline comment.go]: I think this should be `uint` to comply with https://github.com/agilob/go-wypok/pull/13

File: comment.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	if parentCommentId > 0 {
+		methodWithParams += "/" + strconv.Itoa(parentCommentId)
```
Comment (agilob) [inline comment.go]: same here, uint

File: comment.go [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	}
+	return fmt.Sprintf(COMMENTS_BASE_URL, methodWithParams, wh.appKey, wh.authResponse.Userkey)
+}
+
+func getPlusLinkCommentUrl(linkId int, commentId int, wh *WykopHandler) string {
+	return fmt.Sprintf(COMMENTS_BASE_URL, "plus/"+strconv.Itoa(linkId)+"/"+strconv.Itoa(commentId), wh.appKey, wh.authResponse.Userkey)
```
Comment (agilob) [inline comment.go]: and everywhere else

File: comment_test.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+func TestWykopHandler_PlusComment(t *testing.T) {
+	teardownTestCase := setupTestCase(t)
+	defer teardownTestCase(t)
+	wh.LoginToWypok()
+
+	vote, err := wh.PlusComment(1760442, 18549048)
+
+	assert.NotNil(t, vote)
+	assert.Nil(t, err)
+	// Those below are waiting for better times at WykopApi, cause seems that you cannot revoke your "plus" action
```
Comment (agilob) [inline comment_test.go]: In entries to revoke Vote, you just vote again, have you tried this?

Comment (GMolicki) [reply comment_test.go]: Yes I have.
Check comment below assertions.

**Comment by agilob** [comment]
Please update support list in README.md with new functions.

**Pull request merged**